daisy_workflows/image_import/suse/translate.py

```diff
@@ -171,6 +173,31 @@ def _disambiguate_suseconnect_product_error(g, product, error) -> Exception:
     'Unable to find an active SLES subscription. SCC returned: %s' % statuses)
 
 
+def _install_gce_repos(distro, g, include_gce_packages):
+  """Add Zypper repositories that are required for installing GCE packages."""
+  if not include_gce_packages:
+    return
+  # Installing OS Config agent via instructions:
+  #    https://cloud.google.com/compute/docs/manage-os
+  repo = """
+    # Enabled for GCE OS config agent
+    [google-compute-engine]
+    name=Google Compute Engine
+    baseurl=https://packages.cloud.google.com/yum/repos/google-compute-engine-sles{major}-stable
```

Comment:
is this the first time we're adding this repo? i didn't even know it existed. this can bite us later if we ever accidentally upload packages to the repo other than osconfig, bc they may conflict. i think this might be the biggest change in here - have we discussed it with suse?

Reply:
This is the first time using the repo for import.

I like your overall observations; are they limited to this image import usecase? Or are they feedback on the overall OS Config distribution strategy on SUSE and Ubuntu?  (Here's the OS Config setup guide: https://cloud.google.com/compute/docs/manage-os)

Regarding import, I haven't talked with SUSE on this. Not sure about OS config overall.

Reply:
it's more of a risk with import, as we do it instead of the user. if we didn't leave the repo configured after installing packages, there wouldn't be any risk.

Reply:
Talked offline with @hopkiw  and the OS Config team and decided to not install the OS Config agent on SLES, OpenSUSE, or Ubuntu until the agents are supported by upstream. Updated in latest version.